.env

```
@@ -0,0 +1 @@
+const BACKEND_URL = 'http://localhost:3004'
```

👍🏽 nice way to link the backend url 
```suggestion
BACKEND_URL = 'http://localhost:3004'
```

---

Or rename this as REACT_APP_BACKEND_URL

---

renamed: REACT_APP_BACKEND_URL